read_roi/_read_roi.py

```diff
@@ -476,9 +480,13 @@ def read_roi_file(fpath):
 
     logging.debug("Read ROI for \"{}\"".format(name))
 
-    roi, (hdr2Offset, n_coordinates, roi_type, channel, slice, frame, position, version, subtype, size) = extract_basic_roi_data(data)
+    # abb, append group to returned (assigned) tuple
+    #roi, (hdr2Offset, n_coordinates, roi_type, channel, slice, frame, position, version, subtype, size) = extract_basic_roi_data(data)
+    roi, (hdr2Offset, n_coordinates, roi_type, channel, slice, frame, position, version, subtype, size, group) = extract_basic_roi_data(data)
     roi['name'] = name
-
+    # abb, add group to dict
+    roi['group'] = group
+    
```

Comment:
No need for comment describing self-evident operation. Empty lines should not contain spaces.

```suggestion
    roi['group'] = group

```